src/components/layout/nav/PublicMenu.tsx

```diff
@@ -62,7 +61,15 @@ export default function PublicMenu() {
         endIcon={
           open ? <ArrowDropUpIcon color="primary" /> : <ArrowDropDownIcon color="primary" />
         }>
-        <PersonIcon />
+        <Avatar
+          sx={{
+            bgcolor: theme.palette.success.light,
+            height: theme.spacing(4),
+            width: theme.spacing(4),
```

Comment:
You should not be defining width and heights with the spacing function. Hard code them as pixels and make sure to add a media query to make it smaller for smaller screens if needed.